src/App.jsx

```diff
@@ -14,33 +15,87 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const getPrapredGoods = (goods, { sortField, reversed }) => {
+  const preparedGoods = [...goods];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  if (sortField) {
+    preparedGoods.sort((a, b) => {
+      switch (sortField) {
+        case 'alphabet':
+          return a.localeCompare(b);
+        case 'length':
+          return a.length - b.length;
+        default:
+          return 0;
+      }
+    });
+  }
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (reversed) {
+    preparedGoods.reverse();
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  return preparedGoods;
+};
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState(false);
+  const visibleGoods = getPrapredGoods(goodsFromServer, {
+    sortField,
+    reversed,
+  });
+
+  const isResetVisible = sortField !== '' || reversed;
+
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button
```

Comment:
To highlight the active button, you should add an 'is-active' class or similar to visually indicate which sort field is currently active. Right now, you're using 'is-light' to indicate inactive state, but there's no clear active state styling.